File: Bindings~/java/src/main/java/ro/altom/alttester/Notifications/INotificationCallbacks.java
package ro.altom.alttester.Notifications;

public interface INotificationCallbacks {
    void SceneLoadedCallBack(AltLoadSceneNotificationResultParams altLoadSceneNotificationResultParams);
    void SceneUnloadedCallBack(String sceneName);
    void LogCallBack(AltLogNotificationResultParams altLogNotificationResultParams);
    void ApplicationPausedCallBack(boolean applicationPaused);
}

File: Assets/AltTester/Runtime/AltDriver/Notifications/INotificationCallbacks.cs
/*
    Copyright(C) 2026 Altom Consulting
*/


namespace AltTester.AltTesterSDK.Driver.Notifications
{
    public interface INotificationCallbacks
    {
        void SceneLoadedCallback(AltLoadSceneNotificationResultParams altLoadSceneNotificationResultParams);
        void SceneUnloadedCallback(string sceneName);
        void LogCallback(AltLogNotificationResultParams altLogNotificationResultParams);
        void ApplicationPausedCallback(bool applicationPaused);
    }
}
